src/main/java/app/dao/impl/TimesheetDao.java

```diff
@@ -0,0 +1,159 @@
+package app.dao.impl;
+
+import app.entities.Timesheet;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import java.sql.*;
+
+
+public class TimesheetDao {
+    private static final String url = "jdbc:mysql://localhost/timesheet_dev?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC";
+    private static final String user = "root";
+    private static final String password = "root";
+    private static Connection connection;
+    private static PreparedStatement preparedStatement;
+    private static ResultSet resultSet;
+
+
+    public static String findById(int id) {
+        String query = "select * from timesheet where id = ?";
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
```

Comment:
move connection to static method and use it across all functions

Reply:
done

Reply:
connection is taken from class JDBCConection now